<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newHostDialog_q</class>
 <widget class="QDialog" name="newHostDialog_q">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Host</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="titleLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Name the new host object</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>380</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <!-- Page 0: Object name -->
     <widget class="QWidget" name="page_name">
      <layout class="QVBoxLayout" name="page0Layout">
       <item>
        <widget class="QLabel" name="descriptionLabel">
         <property name="text">
          <string>The Host object represents a network node such as a workstation or server with one or several addresses. It is useful when the host has more than one IP address because FirewallFabrik will generate firewall configuration to match all of its addresses. If the host has just one IP address, consider using an Address object instead.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer_name_top">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="nameLayout">
         <item>
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>Name of the new host object:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="obj_name">
           <property name="toolTip">
            <string>Enter the name for the new host. The name must be unique within the library.</string>
           </property>
           <property name="placeholderText">
            <string>Enter host name</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="spacer_name_bottom">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>200</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <!-- Page 1: Configure interfaces -->
     <widget class="QWidget" name="page_interfaces">
      <layout class="QVBoxLayout" name="page1Layout">
       <item>
        <widget class="QLabel" name="ifaceDescriptionLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Add interfaces to the new host. &quot;Name&quot; corresponds to the name of the physical interface, such as &quot;eth0&quot; or &quot;ens192&quot;. &quot;Label&quot; is used to mark an interface to reflect network topology, e.g. &quot;outside&quot; or &quot;inside&quot;. Select &quot;Unnumbered&quot; for interfaces without an IP address (e.g. tunnel endpoints). Select &quot;Dynamic&quot; for interfaces that get their IP via DHCP or PPP.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="interfaceEditor">
         <property name="toolTip">
          <string>Each tab represents one network interface. Use the + and âˆ’ buttons to add or remove interfaces.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ifaceValidationLabel">
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string>color: red;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>161</width>
          <height>37</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="backButton">
        <property name="text">
         <string>&lt; &amp;Back</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextButton">
        <property name="text">
         <string>&amp;Next &gt;</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="finishButton">
        <property name="text">
         <string>&amp;Finish</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>&amp;Cancel</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
